### PR TITLE
Version 1 of Ainneve NPCs

### PR DESCRIPTION
**Pull request by stefanludlow**
This is very much a work in progress, however for other folks working on the Ainneve codebase I imagine having some functional NPCs with behavior demonstrating the gamut of what NPCs can do so far will be useful. This is certainly not a "clean" PR, but as these will very much change over the next few weeks and months while being tested, these should be useful. 

Let me know if we'd like to make any changes to these before merge. Happy to clean out my working notes and such.

### CODE REVIEW

**Comment by feend78** [comment]
So the Travis CI errors in this branch are happening because of some changes to Evennia master. I've pushed an update to the Ainneve master branch to correct them. Please merge the changes from master into your branch and commit them to update this PR and hopefully get a nice happy green checkmark.

**Comment by stefanludlow** [comment]
Yay, passes with flying colors.

File: world/content/sdesc_vars.py [hunk 1]
```diff
@@ -0,0 +1,982 @@
...
+    "grey-striped",
+    "black-eyed",
+    "beady-eyed",
+    "loathesome",
+    "vile",
+    "red-striped",
+    "brown",
+    "black-striped",
+    "dark-green",
+    "thick-carapiced",
```
Comment (feend78) [inline world/content/sdesc_vars.py]: spelling: carapaced

File: world/content/sdesc_vars.py [hunk 2]
```diff
@@ -0,0 +1,982 @@
...
+    "wheat-haired",
+    "henna-haired",
+    "dusky-haired",
+    "ecru-haired",
+    "angular-faced",
+    "aristocratic",
+    "comely-faced",
+    "careworn",
+    "cherubic",
+    "comely-faced",
```
Comment (feend78) [inline world/content/sdesc_vars.py]: comely-faced is duplicate of line 603

File: world/content/sdesc_vars.py [hunk 3]
```diff
@@ -0,0 +1,982 @@
...
+    "saw-toothed",
+    "small-toothed",
+    "hunchbacked",
+    "triangular-faced",
+    "sneer-faced",
+    "gap-toothed",
+    "brown-toothed",
+    "yellow-toothed",
+    "wickedly-fanged",
+]
```
Comment (feend78) [inline world/content/sdesc_vars.py]: PEP 8 formatting likes all files to end with a newline character

**Pull request merged**